var/spack/repos/builtin/packages/eztrace/package.py

```diff
@@ -6,21 +6,69 @@
 from spack import *
 
 
-class Eztrace(AutotoolsPackage):
+class Eztrace(Package):
     """EZTrace is a tool to automatically generate execution traces
        of HPC applications."""
 
-    homepage = "https://eztrace.gforge.inria.fr"
-    url      = "https://gitlab.com/eztrace/eztrace/-/archive/eztrace-1.1-10/eztrace-eztrace-1.1-10.tar.gz"
+    homepage = "https://gitlab.com/eztrace"
     maintainers = ['trahay']
+    git = "https://gitlab.com/eztrace/eztrace.git"
 
-    version('1.1-10', sha256='97aba8f3b3b71e8e2f7ef47e00c262234e27b9cb4a870c85c525317a83a3f0d4')
+    version('1.1-10', tag='eztrace-1.1-10')
+    version('1.1-13', tag='eztrace-1.1-13')
+    version('master',  branch='master')
+    version('2.0-rc1', tag='2.0-rc1')
```

Comment:
Prefer using URL versions rather than tags -- checksumming is safer and it's often faster to download a tarball than do a shallow clone. If you must use tags, always add the git commit SHA as well. Also it's spack convention to list versions in descending order.